src/modules/session/thunks.ts

```diff
@@ -95,6 +97,27 @@ export const createAccount = createAsyncThunk(
   }
 );
 
+/**
+ * Request a iDenfy authToken.
+ * Update state with token and set verified status to pending.
+ */
+export const requestVerificationToken = createAsyncThunk(
+  "session/requestVerificationToken",
+  async (body: IdenfyTokenRequest, { dispatch }) => {
+    const idenfyTokenResponse = await dispatch(
+      idenfyApi.endpoints.requestVerificationToken.initiate(body)
+    );
+
+    if ("error" in idenfyTokenResponse) {
+      return;
+    }
+
+    dispatch(receiveIdenfyToken(idenfyTokenResponse.data.authToken));
+
+    dispatch(updateProfile({ verifiedStatus: "pending" }));
```

Comment:
This is something that has been on my mind. We update status to "pending" after we initiate the verification session. if the user decides not to finish the process, it will remain in pending. They will still be able to create a new session if they login out/in or refresh. So technically this will still work but it would not be an accurate status. @scandycuz  thoughts?

Reply:
@scandycuz After learning more about iDenfy and the process. The piece that i couldn't get around was what you last stated, currently there's no support for adding a callback on token request. 

However, it looks like we will be getting an automatic callback as soon as the user finishes their session. At that point BE will update the status to pending and then approved as soon as they get the second callback. 

We can configure the success/fail url and add a param like you suggested and then send a request to BE for the status, but at that point, why not go with the other solution to send BE a status request every 1 minute or so after triggering a session verification. I can see us having to send interval requests even with the query solution.

I updated the code to remove changing status for now though